WebRoot/h/addattendees

```diff
@@ -92,7 +92,7 @@
 </app:handleError>
 
 <app:view mailbox="${mailbox}" title="${title}" selected='compose' folders="true" tags="true" searches="true" context="${null}" keys="false" date="${requestScope.dateContext}"
-    onload="var e=document.getElementById('findField'); if (e) e.focus();">
+    onload="findField">
```

Comment:
parameter name should be changed to focusEl

Reply:
i doubt if this is needed

Reply:
now because we are not using onload parameter - we can remove it from here (from all files) - what do you suggest?

Reply:
Done